.github/workflows/e2e.yml

```diff
@@ -0,0 +1,20 @@
+name: Testing era_test_node using e2e
+on:
+  workflow_call:
+
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    name: e2e
+    steps:
+      - name: Launch tests
```

Comment:
⚠️ ⚠️ ⚠️ 
Before this step, we need to start the node using artifacts from the build. However, this is part of an effort in another Issue, so will need to wait until that is complete. Once it is, we can do the following step:

```yaml
  - name: Download era_test_node binary
    uses: actions/download-artifact@v2
    with:
      name: era_test_node-binary
      path: path/to/desired/location
  
  - name: Start the era_test_node in the background
    run: |
      era_test_node run &
```

Reply:
I also need to checkout the code 😅 

```yaml
- uses: actions/checkout@v3
  with:
    fetch-depth: 0
```